sdcm/nemesis.py

```diff
@@ -2616,10 +2616,25 @@ def _mgmt_backup(self, backup_specific_tables):
 
     @latency_calculator_decorator(legend="Scylla-Manger repair")
     def disrupt_mgmt_repair_cli(self):
+        if not self.cluster.params.get('use_mgmt') and not self.cluster.params.get('use_cloud_manager'):
+            raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
+        if not self.cluster.scylla_manager_tool.feature("repair", "single-host-parallelism"):
+            self.log.info('single-host-parallelism is not supported running normal repair')
+            self.disrupt_mgmt_repair_cli_normal()
+            return
+
+        match random.choice(["no_no_parallelism", "normal"]):
+            case "no_no_parallelism":
+                self.log.info('single-host-parallelism is supported running repair with single-host-parallelism=1')
+                self._mgmt_repair_cli(host_parallelism=1)
```

Comment:
for longevity tests, 1 sounds too low... could we use something like `len(self.cluster.nodes)/2` or `len(self.cluster.nodes)/3`

Reply:
@fgelcer , from what I understand about this flag is that major use-case for it is to be set to 1, in order to reduce stale repair probability. I guess we could do random from 1 to something. Also take into account that it is not number of nodes participating in the repair, it is maximum number of repairs to run on single node at any time.

Reply:
This part of code is removed